ledger/ledger-api-client/src/it/scala/com/digitalasset/ledger/client/LedgerClientIT.scala

```diff
@@ -75,6 +77,90 @@ final class LedgerClientIT
       }
     }
 
+    "identity provider config" should {
+      val config = IdentityProviderConfig(
+        IdentityProviderId.Id(Ref.LedgerString.assertFromString("abcd")),
+        isDeactivated = false,
+        JwksUrl.assertFromString("http://jwks.some.domain:9999/jwks"),
+        "SomeUser",
+        None, // setting this to any value does not appear to work
```

Comment:
This is unfortunate consequence of sharing the same "domain" model for client and server. This field only makes sense in the latter, it has no meaning in the client space as it is not translated to the wire message.